apps/profiles/models/citylocation.py

```diff
@@ -0,0 +1,64 @@
+from django.contrib.gis.db.models import PointField
+from django.contrib.gis.geos import Point
+from django.db import models
+
+from apps.core.models import AbstractBaseModel
+
+
+class CityLocation(AbstractBaseModel):
+    """
+    Object for storing place data.
+    REST Resources of places from Google MAPS Platform Places API. Response JSON-represented:
+        {
+            "id" : "ChIJ02oeW9PP20YR2XC13VO4YQs"
+            "location" : {
+                "latitude" : 53.902284
+                "longitude" : 27.561831
+            }
+            "viewport" : {
+                "low" : {
+                    "latitude" : 53.82427
+                    "longitude" : 27.38909
+                }
+                "high" : {
+                    "latitude" : 53.97800
+                    "longitude" : 27.76125
+                }
+            }
+        }
+    Resources of places from Yandex Maps Geocode API. Response JSON-represented:
+        {
+          "response": {
+            "GeoObjectCollection": {
+              "featureMember": [
+                {
+                  "GeoObject": {
+                    "metaDataProperty": {...},
+                    "boundedBy": {
+                      "Envelope": {
+                        "lowerCorner": "27.38909, 53.82427",
+                        "upperCorner": "27.76125 53.97800"
+                      }
+                    },
+                    "Point": {
+                      "pos": "27.561831 53.902284"
+                    }
```

Comment:
We can now remove comments abouts apis since it will be frontends responsibility